CharacterCreation/02_Classes.txt

```diff
@@ -427,34 +428,100 @@ Skill Proficiencies:
 * Spotting
 * Survival
 
+== Hunter's Dog ==
+
+    When your character ascends to level 3, you receive an attack dog. Maybe 
+you found the dog along your last mission, or maybe a dog was issued to you when
+you returned to home base (at the GM's choice). Every time you level up after
+you have your dog, your dog levels up as well as defined in the 
+"Class-Specific Documentation/Hunter's_Dog.txt" document. For simplicity, Dogs 
+start at level 3 so that they are always the same level as their owners. If your
+dog is killed in action, you do not get to roll another one. If you and your dog
```

Comment:
I think this makes it sound like you can never have another dog. I think something better might be "you are not automatically issued a replacement" which feels more like you have to work/find a new dog. 
On the other hand, I really like the reminder that your dog is not expendable, and there are consequences to losing it.